src/components/CustomInput.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, TextInput, View } from 'react-native';
+
+interface CustomInputProps {
```

Comment:
dobrą techniką tutaj byłoby przyjęcie przez propsów dokładnie takich samych typów jak komponent `TextInput`. W ten sposób mamy pewność, że inputowi można przekazać jego wszystkie propsy a wyglądałoby to tak:
`type Props = React.ComponentsPropsWithRef<typeof TextInput> & { jakieś nasze propsy, których TextInput nie zawiera, takie jak pewnie label }`